Modules/Core/Common/src/itkExceptionObject.cxx

```diff
@@ -42,7 +42,12 @@ class ExceptionObject::ExceptionData
     , m_Line(line)
   {
     m_What = m_File;
-    m_What += ':' + std::to_string(m_Line) + ":\n";
+    m_What += ':' + std::to_string(m_Line) + ':';
+    if (!m_Location.empty() && m_Location != m_DefaultLocation)
+    {
+      m_What += " in '" + m_Location + "':";
+    }
```

Comment:
Now it avoids appending `" in 'Unknown':"`, when the location is `"Unknown"`. OK?  (We can also remove the condition `m_Location != m_DefaultLocation`, and then just get rid of `m_DefaultLocation` 🤷 )

Reply:
Just removed the condition `m_Location != m_DefaultLocation`!